### DISCUSSION TITLE
Windows laptop or tablet AR

### DISCUSSION THREAD

**Discussion by Giantbean** [google/model-viewer#4856] [Q&A] [answered]
Can model-viewer AR access a web cam on a PC or Laptop? I'm guessing it needs sensors to know the camera position and may not work on a PC or laptop but what about a windows tablet? Can it not work on something like a surface as it uses a standard browser instead of mobile web? 

Edit:
My initial searches found no similar questions, but subsequent searches found [https://github.com/google/model-viewer/discussions/3301](https://github.com/google/model-viewer/discussions/3301) which suggests this is not possible on laptops however I am still unsure of the Windows Tablets. I think the Surface is basically a laptop though so even if it has an accelerometer, it may not have the software to run AR. Can anyone confirm?

**Comment by elalish** [comment, marked as answer]
It probably has the hardware, but it lacks the software. Perhaps if Edge shipped something like ARCore for Windows devices and hooked up the WebXR API to that, but I'm not sure that's a priority.

**Reply by Giantbean** [reply]
I wonder if https://github.com/nativefier/nativefier?tab=readme-ov-file would help it work on PC. If I get a chance to test it I will let you know how it goes.
